releases/sdks/go/gen.yaml

```diff
@@ -7,7 +7,7 @@ generation:
     nameResolutionDec2023: false
   telemetryEnabled: false
 go:
-  version: 0.0.1
+  version: v0.0.0
```

Comment:
speakeasy ?

Reply:
Oh it was overrided by the speakeasy command, let me put that back